magento-research/pwa-studio
# Merge master to develop

## Description

Merges the 3.0 release from `master` into the `develop` branch. The only significant merge conflicts found were in `package.json` files and they were resolved by accepting the higher version numbers in each change set.

## Related Issue

Closes #1389

## Verification Steps

1. Run regression testing
2. Verify no errors

## Screenshots / Screen Captures (if appropriate)

## Checklist:
